### ISSUE TITLE
Update Git-LFS to 2.5.2

### ISSUE THREAD

**Issue by gerhardol** [git-for-windows/git#1849]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

** insert your machine's response here **
```
git version 2.19.0.windows.1
cpu: x86_64
built from commit: d96bb8bc6c636a8869140e860e72e7bdf64bd790
sizeof-long: 4
sizeof-size_t: 8

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

** insert your machine's response here **
```
Microsoft Windows [Version 6.1.7601]
 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** insert your machine's response here **
```
Editor Option: Notepad++
Path Option: Cmd
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: CRLFCommitAsIs
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
Enable Builtin Rebase: Enabled
Enable Builtin Stash: Enabled

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **
BitBucket Server

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** insert your response here **
bash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** insert your commands here **
```
Git-LFS 2.5.1 fails to push to BitBucket server

$ git push
Fatal error: nullects:   0% (0/2), 0 B | 0 B/s
Fatal error: null
error: failed to push some refs to 'ssh://git@stash:7999/gev/repo.git'

 - What did you expect to occur after running these commands?

** insert here **
successful push

 - What actually happened instead?

** insert here **
failure to push

LFS and BitBucket has interpreted the LFS spec differently
https://github.com/git-lfs/git-lfs/issues/3171

One of the following must be done:
 * Update Git-LFS
 * Patch BitBucket Server
 * make a config change

Git-LFS should be updated at next Git release

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** insert URL here **
BitBucket Server is only private

**Comment by shiftkey** [comment]
This has been updated recently at https://github.com/git-for-windows/build-extra/commit/babea8772cddafb2d6e8170481fe5461e9d3c4d2 and will likely be available with the next Git for Windows update.

**Comment by gerhardol** [comment]
Great. If not someone else is before me, I will close when I see the next build.
I expected the update, but want to point out that there is a need and get a reference.

**Comment by dscho** [comment]
@gerhardol have you tested the latest snapshot at https://wingit.blob.core.windows.net/files/index.html? If it already has Git LFS v2.5.2, so will the next Git for Windows version and you can close this ticket.

**Comment by dscho** [comment]
@gerhardol does that thumbs up mean that you tested it and can now confirm that the issue is addressed?

**Comment by dscho** [comment]
@gerhardol so you mean the update to Git LFS v2.5.2 does not fix your issue? Did you open a Git LFS issue at https://github.com/git-lfs/git-lfs?

**Comment by gerhardol** [comment]
> so you mean the update to Git LFS v2.5.2 does not fix your issue?

Correct, LFS-2.5 or later (implicitly git-for-windows 2.19 and later) cannot be used with BitBucket Server.
I support bundling LFS with Git, even if it is not helping me here. (Harder to test using other LFS too.)

Reading the threads about this, the fix in LFS-2.5.2 is fixing a specific issue related to the 2.5 change that some assumed would fix the problem with BitBucket server, but no.
